templates/controllers/grid/settings/sections/form/sectionForm.tpl

```diff
@@ -62,14 +62,22 @@
 		{/fbvFormSection}
 	{/fbvFormArea}
 
-	{fbvFormSection list=true title="user.role.subEditors"}
-		{if count($subeditors)}
-			{foreach from=$subeditors item="subeditor" key="id"}
-				{fbvElement type="checkbox" id="subEditors[]" value=$id checked=in_array($id, $assignedSubeditors) label=$subeditor|escape translate=false}
-			{/foreach}
-		{else}
-			<span class="pkp_form_error"><p>{translate key="manager.section.noSectionEditors"}</p></span>
-		{/if}
+	{fbvFormSection list=true title="manager.sections.form.assignEditors"}
+	<div>{translate key="manager.sections.form.assignEditors.description"}</div>
+	{foreach from=$assignableUserGroups item="assignableUserGroup"}
+		{assign var="role" value=$assignableUserGroup.userGroup->getLocalizedName()}
+		{assign var="userGroupId" value=$assignableUserGroup.userGroup->getId()}
+		{foreach from=$assignableUserGroup.users item=$username key="id"}
+			{fbvElement
+				type="checkbox"
+				id="subEditors[{$userGroupId}][]"
```

Comment:
Missing `|escape` (also for `{translate}` call below) -- this one isn't critical, and probably not the one below either, but a good habit and just in case.